8. Advanced-Quarkus-Modes.md

```diff
@@ -119,4 +119,43 @@ The default command name is `quarkus`, but it can be changed using the property
 mvn clean verify -Dts.quarkus.cli.cmd="java -jar quarkus-cli.jar"
 ```
 
-The above command will use directly the binary from Quarkus upstream build.
\ No newline at end of file
+The above command will use directly the binary from Quarkus upstream build.
+
+# Helm
+
+The framework supports testing the Quarkus Helm extension functionality.
+See the [extension documentation](https://quarkiverse.github.io/quarkiverse-docs/quarkus-helm/dev/index.html).
+
+Running `helm` and `helmfile` CLI is available:
+```java
+@QuarkusScenario
+public class QuarkusHelmFileClientIT {
+
+    private static Path helmfilesFolder;
+
+    @Inject
+    static QuarkusHelmFileClient helmFileClient;
+
+    @BeforeAll
+    public static void tearUp() {
+        helmfilesFolder = Paths.get("src", "test", "resources", "helmfiles").toAbsolutePath();
+    }
+
+    @Test
+    public void verifyHelmFileInjection() {
+        QuarkusHelmFileClient.Result helmCmdResult = helmFileClient.run(helmfilesFolder.toAbsolutePath(), "-version");
+        assertTrue(
+                helmCmdResult.isSuccessful(),
+                String.format("Command %s fails", helmCmdResult.getCommandExecuted()));
+        assertFalse(
+                helmCmdResult.getOutput().isEmpty(),
+                "Unexpected helm version");
+    }
+}
+```
+
+The CLI tools must be installed in the environment.
+
+The default command names are `helm`/`helmfile`. To customize the names, use properties:
```

Comment:
I don't want to be annoying, so take this with reservations BUT - it took me a while that this basically means I can either have aliases for these CLI tools or have them called differently, maybe you could be bit more specific here? e.g. say _By default, Quarkus Test Framework expects [Helm](https://helm.sh/docs/intro/install/) tool available in CLI under `helm`, however if you have the Helm installed under different name, you can take leverage of the `ts.quarkus.helm.cmd` configuration property_

Well, feel free to write it better than I suggested :-)

Reply:
Thanks, great suggestion. I used a slightly modified version of it.